examples/run-command-recipe-org-links.el

```diff
@@ -0,0 +1,64 @@
+
+;; This example demonstrates the use of Elisp for the :command-line argument of
+;; a run-command recipe.
+;; 
+;; Specifically, this recipe creates a temporary org buffer and insert some
+;; headings and some links in it and prompts the user to select one org heading
+;; and then open all links under that heading:
+
+(require 'cl-lib)
+
+(defun run-command-recipe-org-links ()
+  "Prompt the user to select a heading and then open all org links between that
+heading and the next one. If there is no next heading then the search for org
+links will extend to the end of the buffer."
+  (list
+   ;; only enable this command in an org buffer:
+   (when (derived-mode-p 'org-mode)
+     (list :command-name "open heading's link"
```

Comment:
I forgot to mention: `:command-name` is used to generate the command's buffer name, so you'll want something machine-friendly such as `org-links`. You can use `:display` for the user-friendly name.